apis/beacon/states/validators.yaml

```diff
@@ -0,0 +1,47 @@
+get:
+  operationId: "getStateValidators"
+  summary: "Get validators from state"
+  description: "Returns filterable list of validators with their balance, status and index."
+  tags:
+    - Beacon
+  parameters:
+    - name: state_id
+      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/StateId'
+    - name: id
+      description: "Either public key or validator index"
+      in: query
+      required: false
+      schema:
+        type: array
+        maxItems: 30
+        uniqueItems: true
+        items:
+          description: "Hex encoded public key or validator index"
+          type: string
+    - name: status
+      description: "[Validator status specification](https://hackmd.io/ofFJ5gOmQpu1jjHilHbdQQ)"
+      in: query
+      required: false
+      schema:
+        type: array
+        uniqueItems: true
+        items:
+          allOf:
+            - $ref: '../../../beacon-node-oapi.yaml#/components/schemas/ValidatorStatus'
+            - enum: ["active", "pending", "exited", "withdrawal"]
```

Comment:
The other one uses the detailed sub statuses, but this one does not?

Reply:
allOf key combines those two so its effectively extends statuses that can be returned in ValidatorResponse